perl/bin/ascatCounts.pl

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/perl
+
+##########LICENCE##########
+# Copyright (c) 2014-2019 Genome Research Ltd.
+#
+# Author: CASM/Cancer IT <cgphelp@sanger.ac.uk>
+#
+# This file is part of AscatNGS.
+#
+# AscatNGS is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation; either version 3 of the License, or (at your option) any
+# later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+##########LICENCE##########
+
+BEGIN {
+  use Cwd qw(abs_path);
+  use File::Basename;
+  unshift (@INC,dirname(abs_path($0)).'/../lib');
+};
+
+use strict;
+use warnings FATAL => 'all';
+use autodie qw(:all);
+
+use File::Path qw(remove_tree make_path);
+use Getopt::Long;
+use File::Spec;
+use Pod::Usage qw(pod2usage);
+use List::Util qw(first);
+use Const::Fast qw(const);
+use File::Copy;
+use File::Which qw(which);
+use FindBin qw($Bin);
+
+use Sanger::CGP::Ascat::Implement;
+
+use PCAP::Cli;
+
+{
+  my $options = setup();
+  Sanger::CGP::Ascat::Implement::prepare($options);
```

Comment:
Instead of calling this just use the following within the `setup()` function:

```
options->{'tumour_name'} = (PCAP::Bam::sample_name($options->{'tumour'}))[0];
```

This should negate the need to map the tumour bam to the normal, see later comment.